far/FarEng.hlf.m4

```diff
@@ -6179,144 +6179,164 @@ copied.
+ Default value: 0 (allow editing of any files without warnings).
 
- По умолчанию значение = 0.
+ For example, if this parameter is set to #0x43# (binary 0100'0011),
+warning will be shown when opening read-only and system files; editing
+of such files will be disabled.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@. Bits 0 and
+1 are also controlled by the options #Lock editing of read-only files#
+and #Warn when opening read-only files# of the ~Editor~@EditorSettings@
+settings dialog.
 
 
 @Editor.FileSizeLimit
 $ #far:config Editor.FileSizeLimit#
- Параметр "Editor.FileSizeLimit" задаёт максимальный размер редактируемого файла в байтах.
-Если размер редактируемого файла превышает максимально допустимый, то будет показано предупреждающее сообщение перед
-открытием такого файла.
+ This numeric parameter defines the maximum size of a file in bytes
+which can be safely loaded into Editor.
 
- По умолчанию значение = 0 (отключает проверку и вывод сообщения)
+ If the size of a file exceeds this limit, a warning message will
+be shown before opening the file. If the value of this parameter
+is zero, file size check is disabled.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Default value: 0 (file size is not checked, and the warning is never
+displayed).
+
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @System.MsWheelDelta
 $ #far:config System.MsWheelDelta* & System.MsHWheelDelta*#
- Параметры "System.MsWheelDelta*" и "System.MsHWheelDelta*" позволяют менять смещения для
-прокрутки колесом мыши по вертикали и горизонтали.
+ These numeric parameters define scroll speed when mouse wheel is rolled
+or tilted. Scroll speed can be specified separately for different
+directions and different areas.
 
- Параметры для вертикальной прокрутки:
+ Roll the wheel one notch to scroll the specified number of lines at
+a time vertically:
 
- System.MsWheelDeltaView  - в программе просмотра
- System.MsWheelDeltaEdit  - во встроенном редакторе
- System.MsWheelDeltaHelp  - в системе помощи
- System.MsWheelDelta      - в прочих областях
+ System.MsWheelDeltaView  - ^<wrap>in the internal Viewer
+ System.MsWheelDeltaEdit  - in the internal Editor
+ System.MsWheelDeltaHelp  - on help pages
+ System.MsWheelDelta      - in other areas
 
- Параметры для горизонтальной прокрутки (Windows Vista и выше):
+ Tilt the wheel to scroll the specified number of characters at a time
+horizontally (Windows Vista and above):
 
- System.MsHWheelDeltaView - в программе просмотра
- System.MsHWheelDeltaEdit - во встроенном редакторе
- System.MsHWheelDeltaHelp - в системе помощи
- System.MsHWheelDelta     - в прочих областях
+ System.MsHWheelDeltaView - ^<wrap>in the internal Viewer
+ System.MsHWheelDeltaEdit - in the internal Editor
+ System.MsHWheelDelta     - in other areas
 
- По умолчанию значение = 1
+ Default value: 1 (for all parameters).
 
```

Comment:
Need to add note that `Alt+Wheel` always scrolls one line / character at at time.